examples/basic-iframe/.tina/database.js

```diff
@@ -15,67 +15,66 @@ const octokit = new Octokit({
   auth: token,
 })
 
-const level = new ManyLevelGuest()
-const socket = connect(9000)
-
-// Pipe socket into guest stream and vice versa
-pipeline(socket, level.createRpcStream(), socket, () => {
-  // Disconnected
-})
+const localLevelStore = new TinaLocalLevel()
 
 export default createDatabase({
   bridge: new FilesystemBridge(process.cwd()),
-  level,
+  level: localLevelStore,
   onPut: async (key, value) => {
-    let sha
-    try {
-      const {
-        data: { sha: existingSha },
-      } = await octokit.repos.getContent({
-        owner,
-        repo,
-        path: key,
-        branch,
-      })
-      sha = existingSha
-    } catch (e) {}
+    const filePath = path.join(process.cwd(), key)
```

Comment:
did you mean to change these? if you are specifying the filesystem bridge, its already going to be updating the local fs, so you could just leave onPut/onDelete out here

Reply:
When I was running the code I was getting errors from Github (404 errors) when saving. When I swapped out onput and onDelete it worked.

Reply:
yeah, owner/repo/token/branch need to be configured for onPut/onDelete to work